homeassistant/components/withings/sensor.py

```diff
@@ -259,6 +259,13 @@ class WithingsMeasurementSensorEntityDescription(SensorEntityDescription):
         native_unit_of_measurement=PERCENTAGE,
         entity_registry_enabled_default=False,
     ),
+    MeasurementType.BASAL_METABOLIC_RATE: WithingsMeasurementSensorEntityDescription(
+        key="basal_metabolic_rate",
+        measurement_type=MeasurementType.BASAL_METABOLIC_RATE,
+        translation_key="basal_metabolic_rate",
+        native_unit_of_measurement="kcal",
```

Comment:
Does this need to a unit measured over some time period? (I don't know much about this, just reading wikipedia)

Reply:
That's the same as the state class right? Or do you mean something else? (You probably do, otherwise you wouldn't create 2 comments)

But yea, I should add a state class

Reply:
I was intending to make a different comment -- but maybe its the same depending on which state class. I'm not a units expert though.  My impression is that basal metabolic rate is a unit of energy over time. My impression is that kcal is a unit of energy not over time.

I was assuming the state class is 'measurement' but if itsof the total ones then I agree its just the state class thing since then the rate over time will be implied by the total increasing over time.

Reply:
Oh check, good point, I'll have a double check on that

Reply:
cc @anthonym21

How does Withings display this? Is this the value per day but not annotated as `/day`? Maybe we should add it here to be correct and complete

Reply:
noted in other comment but it is intended to represent a single day's calorie expenditure i.e. kcal/day - note that if the user weighs themselves multiple times a day then it will change multiple times a day, but I think that's alright here with the device = energy and state = measurement as I just tried it out and it shows a range for the day in the graph and frankly that's good enough for this metric, which is at best a contrived estimation.